### ISSUE TITLE
Move import otter below h1 if present

### ISSUE THREAD

**Issue by pancakereport** [ucbds-infra/otter-grader#985]
**Is your feature request related to a problem? Please describe.**
Berkeley is now subject to [additional digital accessibility requirements](https://dap.berkeley.edu/ada-title-ii-updates/ada-title-ii-updates-what-does-it-mean-us). We are using https://github.com/berkeley-dsep-infra/jupyterlab-a11y-checker to ensure Jupyter Notebook accessibility. Assignments that are created with otter are raising concerns because the `import otter` cell is placed above a level one header (if it exisits). It's a little unclear to me personally if this is an [accessibility violation](https://docs.google.com/spreadsheets/d/1TBE4BhcmAN2wQHGYSvRGGdyL2pWTVw4cLFU3mARMfjM/edit?gid=0#gid=0) (row 36) or just a problem with [best practice](https://dequeuniversity.com/rules/axe/3.5/heading-order). Either way, it would be awesome if this could be addressed upstream by otter itself. I can definitely look into this if needed, I'm just not super familiar with the code base at this time. 

**Describe the solution you'd like**
If h1 is present at the top of the notebook passed to `otter assign`, put `import otter` code cell below the first markdown cell in the notebook.

If the first cell in the notebook is not markdown/h1 is not present at the top of the notebook, `import otter` can remain the first cell of the generated notebook.

**Describe alternatives you've considered**
We could edit the student notebook after it's generated, but this is annoying and I dislike adding manual steps after notebook generation.
